src/Service/CodeCommit/tests/Unit/CodeCommitClientTest.php

```diff
@@ -47,7 +50,7 @@ public function testGetDifferences(): void
         $client = new CodeCommitClient([], new NullProvider(), new MockHttpClient());
 
         $input = new GetDifferencesInput([
-            'repositoryName' => 'change me',
```

Comment:
decided to change this from the lazily-added default while i was here

Reply:
thank you :heart: